### PR TITLE
Added reuse information

### PR DESCRIPTION
**Pull request by DerMolly**
- [x] reuse lint in lint workflow
- [x] added reuse information to all files

### CODE REVIEW

**Comment by mrdrogdrog** [comment]
Please create a new workflow for this.

**Comment by mrdrogdrog** [comment]
This way you can see if code linting of license linting fails on GitHub.

Another reason is to keep the workflows simple and they can run in parallel.

**Comment by mrdrogdrog** [comment]
Did you check if there is already a github actions for that? If not, is it possible to cache the python package?

**Comment by davidmehren** [comment]
There is https://github.com/marketplace/actions/reuse-compliance-check

**Comment by DerMolly** [comment]
done

File: LICENSES/LicenseRef-HedgeDoc-Icon-Usgae-Guidlines.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# HedgeDoc Logo Usage Guidelines
```
Comment (ErikMichelson) [inline LICENSES/LicenseRef-HedgeDoc-Icon-Usgae-Guidlines.txt:1]: Filename has typo

File: craco.config.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: CC0-1.0
```
Comment (ErikMichelson) [inline craco.config.js:4]: Why is this CC0 instead of AGPL3.0-only?

Comment (DerMolly) [reply craco.config.js:4]: I put all config files as CC0

Comment (mrdrogdrog) [reply craco.config.js:4]: Why

File: LICENSES/LicenseRef-HedgeDoc-Icon-Usgae-Guidlines.txt (hunk 1 above)
Comment (DerMolly) [reply LICENSES/LicenseRef-HedgeDoc-Icon-Usgae-Guidlines.txt]: fixed

File: craco.config.js (hunk 2 above)
Comment (DerMolly) [reply craco.config.js:4]: This was suggested by https://reuse.software/tutorial/#insignificant-files

Comment (ErikMichelson) [reply craco.config.js:4]: For files like config.json, .renovaterc.json etc that indeed makes sense, but I'd see this more as a source code file than a config file.

Comment (DerMolly) [reply craco.config.js:4]: okay anything else I should put under AGPLv3

File: src/components/editor/editor-pane/one-dark.scss [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
- * Atom One Dark Theme
- *
+/*
  * Copyright (c) 2015 Hikio - twitter.com/hik_io
+ * SPDX-FileCopyrightText: 2015 Hikio - twitter.com/hik_io
  *
- * 06/26/2015
- *
- * Licensed under MIT
- * GitHub https://github.com/hikio/brackets-one-dark
```
Comment (ErikMichelson) [inline src/components/editor/editor-pane/one-dark.scss:9]: Maybe it's useful to keep the GitHub repo and the theme name somewhere around.

File: craco.config.js (hunk 2 above)
Comment (DerMolly) [reply craco.config.js]: this and package.json changed

File: src/components/editor/editor-pane/one-dark.scss (hunk 3 above)
Comment (DerMolly) [reply src/components/editor/editor-pane/one-dark.scss]: fixed

**Review by mrdrogdrog — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**